app/services/wikipedia_linking_service.py

```
@@ -0,0 +1,733 @@
+            except json.JSONDecodeError as e:
+                logger.error(f"Failed to parse LLM response as JSON: {e}")
+                logger.debug(f"LLM response: {response_content[:200]}...")
+                
+                # Try to fix common JSON issues
+                try:
+                    # Remove any trailing commas or fix quotes
+                    cleaned_response = response_content.replace("'", '"')
+                    search_terms = json.loads(cleaned_response)
+                    logger.info("Successfully parsed JSON after cleaning")
+                    return search_terms
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)